nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -52,15 +52,11 @@
     NeuralType,
     SpectrogramType,
 )
-from nemo.utils import logging
+from nemo.utils import logging, model_utils
 
 try:
     from sacrebleu import corpus_bleu
 
-    from nemo.collections.nlp.modules.common import TokenClassifier
-    from nemo.collections.nlp.modules.common.lm_utils import get_transformer
-    from nemo.collections.nlp.modules.common.transformer import BeamSearchSequenceGenerator, TransformerEncoder
-
     NLP_AVAILABLE = True
 except (ImportError, ModuleNotFoundError):
     NLP_AVAILABLE = False
```

Comment:
remove this NLP import check after we switch over to ASR only dependency @stevehuang52

Reply:
done